src/main/webapp/resources/js/pages/analysis/components/AnalysisTextPreview.jsx

```diff
@@ -108,7 +105,7 @@ export default function AnalysisTextPreview({ output }) {
           </TextOutputWrapper>
           <div>
             <Space direction={"horizontal"}>
-              <span id={`${output.filename}-preview-status`}></span>
+              <span id={`${output.filename}-preview-status`}>{fileSizePreview}</span>
```

Comment:
**praise:** This is a great way to handle it, not really sure why it wasn't set up like this to begin with.